contrib/test/ci/crio.service

```diff
@@ -1,16 +1,16 @@
 [Unit]
-Description=Open Container Initiative Daemon
-Documentation=https://github.com/kubernetes-sigs/cri-o
-After=network-online.target
+Description=Container Runtime Interface for OCI (CRI-O)
+Documentation=https://github.com/cri-o/cri-o
+Wants=network-online.target remote-fs.target
+After=network-online.target remote-fs.target
 
 [Service]
 Type=notify
 EnvironmentFile=-/etc/sysconfig/crio
-EnvironmentFile=-/etc/sysconfig/crio-metrics
-EnvironmentFile=-/etc/sysconfig/crio-network
-EnvironmentFile=-/etc/sysconfig/crio-storage
 Environment=GOTRACEBACK=crash
```

Comment:
why remove this?

Reply:
Shall this file sync with https://github.com/cri-o/cri-o/pull/4128/files#diff-8f149e7773eb6fec135aa90b23aac30b?